products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,178 @@
-# Global Documentation Search
+# Documentation Search MVP
 
-Author: Bill Fienberg
-Last Updated: 2020-01-23
-Reviewers: TBD
+**Author(s):** Bill Fienberg <bill.fienberg@oddball.io>
 
-## Objective
+**Last Updated:** 2020-02-11
 
-## Background
+**Status:** **Draft**
+
+**Approvers:** Andrew G., Megan K., Rian F., Wyatt W.
 
 ## Overview
 
-## Detailed Design
+### Objective
+
+Add search functionality to a public documentation landing page
+
+### Background
+
+The documentation we have is spread across many different locations (public repos, private repos, GitHub pages, handbooks, etc). There currently isn't a way for VFS/VSP users to search multiple documentation sources with a single query.
+
+### High Level Design
+
+We are adding a public landing page that will contain a search input where users can search multiple sources of documentation with one query.
+
+## Specifics
+
+### Detailed Design
+
+For a multi-repo search MVP, we are adding an HTML text input and [Algolia's DocSearch JS snippet](https://github.com/algolia/docsearch) to the [`va.gov-team` GitHub Pages site](https://department-of-veterans-affairs.github.io/va.gov-team/), [Algolia's Search API](https://www.algolia.com/products/search/), and [Algolia's scraper](https://github.com/algolia/docsearch-scraper).
+
+In general, this kind of system requires the following components:
+
+- Page with Input Textbox - We need a public page with a search input where users can input their search term(s).
+- Index/Database - We need a place to store the records that will be returned to the user.
+- Search API - We need a way to get data out of the index.
+- Crawler/Scraper - We need a crawler to scan our documentation sources and push data to our index.
+- Pipeline - We need a way to routinely run the crawler
+
+Each one of the above components could be a separate buy-or-build decision.
+
+By leveraging the following existing technologies, we should only need to add the search input to the landing page, configure the scraper, and configure when to run the scraper.
+
+- Algolia's Search API product will satisfy the index and API components.
```

Comment:
Are we missing the database portion, or is that what you mean by `index` here? If so, could you call that out more specifically and link to the page on Algolia's site that explains it? (This is the part that we'd have to pay for, right?)